src/coreclr/vm/i386/gmsx86.cpp

```diff
@@ -827,6 +827,8 @@ void LazyMachState::unwindLazyState(LazyMachState* baseState,
             case 0x89:                          // MOV r/m, reg
                 if (ip[1] == 0xEC)              // MOV ESP, EBP
                     goto mov_esp_ebp;
+                if (ip[1] == 0xDC)              // MOV ESP, EBP
```

Comment:
Is "MOV ESP, EBP" correct?

Reply:
No. I'll update the comment when I finish this patch. This still doesn't solve the whole issue, but I got roped into some other issues the past couple of weeks. Thanks for noting this.